### PR TITLE
CI-4085 refactoring the docs to detail the new 6.5 and skyline merged support

### PR DESCRIPTION
**Pull request by francoisledroff**

### CODE REVIEW

File: using/aem-event-setup.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 # Setting up AEM Events with Adobe I/O Events
 
-These instructions describe how to set up Adobe Experience Manager (AEM) for Adobe I/O Events. You can use Adobe I/O for notification of AEM events, such as page or asset changes.
+These instructions describe how to set up Adobe Experience Manager (AEM) for Adobe I/O Events. 
+You can use Adobe I/O for notification of AEM events, such as page or asset changes.
```
Comment (kgera) [inline using/aem-event-setup.md:6]: These instructions describe how to set up Adobe Experience Manager (AEM) to integrate with Adobe I/O Events. 
You can use Adobe I/O Events for notification of AEM events, such as page or asset CRUD operations.

File: aem/aem_skyline_install.md [hunk 2]
```diff
@@ -3,80 +3,33 @@
...
+   
+Once the previous steps are completed, a new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console. 
+Please refer to the Developer Console documentation on how to [add Events to a project](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-add-event.md) 
+if you are new to this.
+ 
+  ![An Adobe I/O Console `AEM as a Cloud Service` Events Provider](../img/add_skyline_event_provider.png "An Adobe I/O Console `AEM as a Cloud Service` Events Provider")
+ 
 
-### Ping Events 
+### Checks events emitting
```
Comment (kgera) [inline aem/aem_skyline_install.md:32]: Verify Events are being Emitted from 'AEM as a Cloud Service'

File: aem/aem_skyline_install.md [hunk 3]
```diff
@@ -3,80 +3,33 @@
...
-5. Now you should see this new Adobe I/O Events IMS Configuration, and you can select it to check its health.
-
-     ![Adobe IMS Configuration IMS Health Check](../img/events_aem_adobe-ims-conf-4.png "Adobe IMS Configuration Health Check")
-      
+3. [Deploy the code](https://docs.adobe.com/content/help/en/experience-manager-cloud-manager/using/how-to-use/deploying-code.html) 
+3. [Finalize the Adobe IMS configuration in AEM](aem_ims_config.md)
+   
+### Verify your AEM Cloud Service is showing in the Adobe I/O Developer console
+   
+Once the previous steps are completed, a new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console.
```
Comment (kgera) [inline aem/aem_skyline_install.md:25]: Once the previous steps are completed, a new `AEM as a Cloud Service` provider should appear in the  `Events Providers` list in the Adobe I/O developer console.

File: aem/aem_skyline_install.md [hunk 4]
```diff
@@ -3,80 +3,33 @@
...
+3. [Finalize the Adobe IMS configuration in AEM](aem_ims_config.md)
+   
+### Verify your AEM Cloud Service is showing in the Adobe I/O Developer console
+   
+Once the previous steps are completed, a new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console. 
+Please refer to the Developer Console documentation on how to [add Events to a project](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-add-event.md) 
+if you are new to this.
+ 
+  ![An Adobe I/O Console `AEM as a Cloud Service` Events Provider](../img/add_skyline_event_provider.png "An Adobe I/O Console `AEM as a Cloud Service` Events Provider")
+
```
Comment (kgera) [inline aem/aem_skyline_install.md:30]: An Adobe I/O developer Console `AEM as a Cloud Service` Events Provider

**Review by kgera — CHANGES_REQUESTED** [review]
Please update I/O Console to I/O developer console.

File: aem/aem_advanced_configurations.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+When an OSGI event of interest is triggered (i.e an OSGI events that matches one of your `OSGI to XDM event mapping configurations`), it makes it to a job queue handled by `Sling Job Handler`.
+
+This job is persisted in the resource tree (for failover etc.), then the job is distributed to an instance responsible for processing the job; and on that instance the job is put into a processing queue, where eventually a `Sling Job Consumer` will execute it.
+
+Adobe I/O Events `Sling Job Consumer`'s job is to send this event to Adobe I/O.
+* `JobResult.OK` should be returned. If the job has not been processed completely,
+* `JobResult.FAILED` should be returned if Adobe I/O fails either to receive or to process it (due to network failure or Adobe I/O failure).
+In that case the job will be rescheduled/retried   
+* if the max number of retries is met, the process will not be rescheduled and treated like the method would have returned `JobResult.CANCEL`.
```
Comment (shikhartanwar) [inline aem/aem_advanced_configurations.md:45]: Do we really need this section? There's nothing the end developer can do here.

File: aem/aem_console_setup.md [hunk 6]
```diff
@@ -1,36 +1,76 @@
 
 
-### Add the `I/O Events` API to your Adobe developer console project
-
-
-For your AEM to be authorized to emit events against Adobe I/O, you need to add the `I/O Events` API to your project in the Adobe developer console. If you are new to this, please refer to the [step-by-step instructions for creating an empty project in Adobe Developer Console](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/projects-empty.md) )
+### Add the `I/O Events` and `I/O Management API` API to your Adobe developer console workspace
```
Comment (shikhartanwar) [inline aem/aem_console_setup.md:3]: Why both? 
I/O Management API -> For registering the provider using v2 APIs
I/O Events API -> For emitting events
Is the above understanding correct?

File: using/aem-event-setup.md (hunk 1 above)
Comment (shikhartanwar) [reply using/aem-event-setup.md:6]: CRUD operations -> CUD operations

File: aem/aem_on_premise_install_6.5.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+
+## Integrate with AEM 6.5.x
+
+This documentation is specific to `AEM on premise` version 6.5.x, 
+to integrate with `AEM as cloud service`, please refer to our [`AEM as cloud service` documentation](aem_skyline_install.md).
+
+To install and configure the AEM event proxy package on version 6.5.x:
+
+1. Download the latest version of the package [version 6.5.6](https://github.com/AdobeDocs/adobeio-events/files/3729022/aem-event-proxy-6.5.6.zip)
```
Comment (shikhartanwar) [inline aem/aem_on_premise_install_6.5.md:9]: This is not the latest merged version right. We should add link to the latest version here.

File: aem/aem_console_setup.md (hunk 6 above)
Comment (francoisledroff) [reply aem/aem_console_setup.md:3]: yes we need both

File: aem/aem_on_premise_install_6.5.md (hunk 7 above)
Comment (francoisledroff) [reply aem/aem_on_premise_install_6.5.md:9]: yes, will do

File: aem/aem_advanced_configurations.md (hunk 5 above)
Comment (francoisledroff) [reply aem/aem_advanced_configurations.md:45]: This was in the doc before... I did not want to delete it.. found it useful

**Comment by francoisledroff** [comment]
> Please update I/O Console to I/O developer console.

done

**Review by kgera — APPROVED** [review]

**Comment by kgera** [comment]
Looks good to me now. Can be merged @francoisledroff

**Pull request merged**